Documentation/gettingstarted/dns.rst

```diff
@@ -0,0 +1,127 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+****************************************
+Getting Started Using DNS-Based Policies
+****************************************
+
+This document serves as an introduction for using Cilium to enforce DNS-based
+security policies for Kubernetes pods.
+
+.. include:: gsg_intro.rst
+.. include:: minikube_intro.rst
+.. include:: cilium_install.rst
+
+Step 2: Deploy the Demo Application
+===================================
+
+DNS-based policies are very useful for controlling access to services running outside the Kubernetes cluster. Whether it is cloud services such as S3, DynamoDB, RDS, etc. or private VPC services outside the cluster, DNS provides persistent identity for these external services. CIDR or IP-based policies are cumbersome and hard to maintain as the IPs associated with external services can change frequently. The Cilium DNS-based policies provide an easy mechanism to specify access control while leaving the harder aspects of resolving and enforcing IP-based filters to Cilium. 
```

Comment:
The intro is still a bit misleading as it seems to suggest to be AWS specific. I'm also not sure what "private VPC services" are.

You should reword the "resolving and enforcing IP-based filters" as it suggests polling. I would keep this universal such as: Keeping track of DNS name to IP address mapping.